cf-cnb-clusterbuildtemplate.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: ClusterBuildTemplate
+metadata:
+  name: cf-cnb
```

Comment:
I think we should we call this `riff-application-cnb` and then rename the current `riff-cnb` to `riff-function-cnb`.

Reply:
I'll rename `cf-cnb-*` to `riff-app-cnb*`. I don't want to change `riff-cnb-*` since that would be a breaking change with the current riff cli.

Reply:
bumped the version and renamed the clusterbuildtemplates to riff-application and riff-function